test/show.jl

```diff
@@ -3,6 +3,7 @@
     @test sprint(show, (x*y^2 + x + 1 + y)) == "xy² + x + y + 1"
     @test sprint(show, (x + 1 + y) / x^2) == "(x + y + 1) / (x²)"
     @test sprint(show, (x - y - x + y) / (x^2 - x)) == "(0) / (x² - x)"
+    @test sprint(show, CustomPoly(1 + x)) == "CustomPoly{Int64,DynamicPolynomials.Polynomial{true,Int64}}(x + 1)"
```

Comment:
The tests are also run for TypedPolynomials :)

Reply:
Oups :-P